### ISSUE TITLE
Not able to unblock multiple users and stop sending notification when they are block ios

### ISSUE THREAD

**Issue by ZalakSA** [QuickBlox/quickblox-ios-sdk#997]
Code for sending push notification is as below:

```
 let aps = ["SA_STR_PUSH_NOTIFICATION_DIALOG_ID".localized: "\(self.dialog.id!)","user_id" : "\(self.dialog.recipientID)","name": ServicesManager.instance().currentUser.fullName,"ios_sound": "default",QBMPushMessageSoundKey: "default", "notificationType" : "chat" ,"save_to_history" : true , "ios_badge": "1", "message" : "\(message.text!)"] as [String : Any]
                
                do {
                    
                    let jsonResult = try JSONSerialization.data(withJSONObject: aps, options: JSONSerialization.WritingOptions.prettyPrinted)
                    let messageTosend = String(data: jsonResult, encoding: .utf8)
                    
                    let event = QBMEvent()
                    event.notificationType = .push
                    event.usersIDs = "\(self.dialog.recipientID)"
                    event.type = .oneShot
                    event.message = messageTosend
                    
                    QBRequest.createEvent(event, successBlock: { (response,event) in
                        print(response)
                    }) { (error) in
                        print(error)
                    }
                }catch{
                    
                }

```
To block user I have used below method: 

```
let item: QBPrivacyItem = QBPrivacyItem.init(privacyType: QBPrivacyType.userID, userID: UInt(userQbID)!, allow: allow)!
        
        item.mutualBlock = true
        if APP_DELEGATE.privacyList != nil {
            if !allow
            {
                APP_DELEGATE.privacyList.addObject(item)
            }
            QBChat.instance.setPrivacyList(APP_DELEGATE.privacyList)
            QBChat.instance.setDefaultPrivacyListWithName(publicTxt)
        }
        else {
            let list: QBPrivacyList = QBPrivacyList(name: publicTxt, items: [item])
            QBChat.instance.setPrivacyList(list)
            QBChat.instance.setDefaultPrivacyListWithName(publicTxt)
        }
```
        
To block pass value as "false" to allow variable while creating QBPrivacyItem and "true" for unblocking

I am able to block multiple users but I am not able to unblock because I am not able to remove item from privacy list. 

And User is blocked still push notification appears, how to stop coming notification from quick blox when user is blocked

Please help me to resolve this issue.

Thanks,

**Comment by ZalakSA** [comment]
@Raikerian 

I have already created separate issue with details.

Can you please go through it and if extra information is needed, I will for sure provide it. Because I need your help.

Thanks,

**Comment by ZalakSA** [comment]
@VitaliyGuru Hello, Do you need more details for my issue?

Please help me for this issue.

Thanks

**Comment by VitaliyGuru** [comment]
@ZalakSA, Please follow [these instructions](https://github.com/QuickBlox/quickblox-ios-sdk/issues/948#issuecomment-373396277).

**Comment by pro100andrey** [comment]
Hi @ZalakSA any updates?

**Comment by pro100andrey** [comment]
No activity. Closing.
